src/event-side-effects.ts

```diff
@@ -0,0 +1,100 @@
+export const gameButton= document.getElementById("simpleGame") as HTMLButtonElement;
+export const connectorButton= document.getElementById("personalInfo") as HTMLButtonElement;
+export const gradientColor = document.getElementById("changeClass") as HTMLButtonElement;
+//export const cloneLastButton = document.getElementById("cloneLastCard") as HTMLButtonElement;
+export const cloneCardButton = document.getElementById("cloneCard") as HTMLButtonElement;
+export function chooseNumber(): void {
+    const drawTicket = Math.floor((Math.random() * 100) + 1);
+    let number;
+    let howMany = 0;
+
+    do {
+        number = parseInt(prompt("Podaj liczbe !"));
+        howMany = howMany + 1
+
+        if (number > drawTicket) {
+            console.log("To za duzo")
+        } else if (number < drawTicket) {
+            console.log("To za malo")
+        } else if (number == drawTicket) {
+            console.log("Wygraleś")
+            let dec = prompt("Czy chcesz zagrac jeszcze raz ? T/N");
+            if (dec == "T") {
+                howMany = 0;
+                continue;
+            } else {
+                break;
+            }
+        }
+
+        if (howMany == 20 && number != drawTicket) {
+            console.log("Przegrales")
+            let dec = prompt("Czy chcesz zagrac jeszcze raz ? T/N");
+            if (dec == "T") {
+                howMany = 0;
+            } else {
+                break;
+            }
+        }
+    } while(number != drawTicket)
+}
+export function cursorOnButton(){
+    console.log("Cursor on the button");
+}
+export function changeColor(){
+    gameButton.style.background = "#79462C";
+}
+export function getPersonalData(){
+    let firstName = prompt("Type your First Name!");
+    let lastName = prompt("Type your Last Name!");
+    console.log(`Hello ${firstName} ${lastName}!`)
+
+}
+export function gradientColorChange(){
+    gradientColor.className ="gradientChange";
+}
+export function changeCSSClass() {
+    document.getElementById("simpleGame").className = "btn btn-danger";
+}
+export function backToNormal() {
+    document.getElementById("simpleGame").className = "btn btn-secondary"
+}
+//export function cloneCard() {
+//
+//    const node = document.getElementById("lastRow");
+//    const clone = node.children;
+//    const singleCard = clone[0];
+//    const cardCopy = singleCard.cloneNode(true);
+//
+//    document.getElementById("lastRow").appendChild(cardCopy);
+//}
+//#1 Funkcja która pobiera informacje o tym, który card chcemy powielić
+export function chooseCardToClone() {
+    const cardNumber = parseInt(prompt("Select the number of a card you want to clone!"));
+
+}
+//#2 Funkcja która zaznacza children z podanego id
+export function getChildrenById(){
+    const listOfCards = document.getElementById("cards");
+    const cardChildren = listOfCards.children;
```

Comment:
if we want to get the result of a function we need to define the result in `return`